frontend/plugins/frontline_ui/src/modules/forms/components/FormEdit.tsx

```
@@ -0,0 +1,13 @@
+import { useParams } from 'react-router-dom';
+import { useFormDetail } from '../hooks/useFormDetail';
+import { FormSetupSteps } from './FormSetupSteps';
+import { Spinner } from 'erxes-ui';
+
+export const FormEdit = () => {
+  const { formId } = useParams();
+  const { formDetail, loading } = useFormDetail({ formId: formId || '' });
+
+  if (true) return <Spinner />;
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)